src/components/PlantingStep/PlantingStep.tsx

```diff
@@ -1,25 +1,36 @@
-import React from 'react';
+import React, { memo } from 'react';
 import { View, Text, TouchableOpacity } from 'react-native';
 import { useStyles } from 'react-native-unistyles';
 import { useTranslation } from 'react-i18next';
 
-import { PlantingStep as PlantingStepType } from '@src/api/model';
+import { PlantingStep as TPlantingStepType } from '@src/api/model';
 import { styleSheet } from './PlantingStep.style';
+import { TStapDay } from '../PlantCareBody/PlantCareBody';
 
-type Props = PlantingStepType & {};
+type Props = {
+	stepDay: TPlantingStepType;
+	handlerCurStepDay: (day: TStapDay) => void;
```

Comment:
are TPlantingStepType and TStapDay the same type?

Reply:
no